src/head.js

```diff
@@ -0,0 +1,7 @@
+const headArray = (array) =>{
+for (let i=0; i < array.length; i++) {
+  return array[i]
+}
+// return array[0]
```

Comment:
Also if you are returning the first item in the array this for loop is redundant, so you don't need it: 

```
return array[0]
```